### DISCUSSION TITLE
Integrated Developer Documentation Hosting

### DISCUSSION THREAD

**Discussion by builtcuriously** [heyform/heyform#62] [Ideas]
The ability to host and manage the developer documentation directly within the HeyForm repository would allow self-hosters and contributors to create, update, and access documentation alongside the codebase. 

## Benefits
- locally accessibility 
- versioned to last sync to origin 
- community contributions 

## Implementation Suggestion
A separate directory within the monorepo such as `/packages/docs`. The existing docs are in Notion markdown and thus formatted appropriately.

Thank you for considering this suggestion ☺️

**Comment by iMuFeng** [comment]
We will soon be migrating our documents from Notion to GitHub.

**Comment by dearroy** [comment]
@builtcuriously This has been completed, check the new [documentation](https://docs.heyform.net) and its [public repo](https://github.com/heyform/docs).
